projects/questionset-editor-library/src/lib/components/options/options.component.ts

```diff
@@ -109,7 +109,7 @@ export class OptionsComponent implements OnInit, OnChanges {
       templateId: this.templateType,
       name: this.questionPrimaryCategory || 'Multiple Choice Question',
       responseDeclaration: this.getResponseDeclaration(editorState),
-      outcomeDeclaration: this.getOutcomeDeclaration(),
+      outcomeDeclaration: this.getOutcomeDeclaration(), 
```

Comment:
@pavankumar0408 Please remove this unnecessary spacing change it might impact the PR code coverage.